paper_demo/xpcs_client.py

```diff
@@ -26,12 +26,8 @@ class XPCSBoost(GladierBaseClient):
     globus_group = '368beb47-c9c5-11e9-b455-0efb3ba9a670'
     gladier_tools = [
         "gladier_tools.globus.transfer.Transfer:FromStorage",
-#        PrePublish,
-        AcquireNodes,
```

Comment:
Should AcquireNodes be kept?

Reply:
I didn't think it added much value here. We added it to help Suresh understand where flows were slow, but chances are anyone running this 1) will have a dedicated endpoint, and 2) won't mind if the analysis step includes queue time.

Reply:
Sounds good